### PR TITLE
events are reading.

### PR DESCRIPTION
**Pull request by acumberlander**

### CODE REVIEW

File: src/components/Pages/Friends/friends.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import './friends.scss';
-// import { homedir } from 'os';
```
Comment (morecallan) [inline src/components/Pages/Friends/friends.js:3]: Make sure to completely remove commented out or unused code

**Review by morecallan — COMMENTED** [review]
One small thing here but overall looks good! Please push up the change and I’ll merge.

File: src/components/Pages/Friends/friends.js (hunk 1 above)
Comment (acumberlander) [reply src/components/Pages/Friends/friends.js:3]: Commented code is gone now

**Comment by morecallan** [comment]
👍

**Pull request merged**